README.md

```diff
@@ -275,6 +275,9 @@ end
 ```
 
 ### [oneview_logical_interconnect](examples/logical_interconnect.rb)
+
+Note: if you are using API1200 and onwards, see examples[here](examples/logical_interconnect_api_1200.rb).
```

Comment:
Keep a space between examples and [here].

Reply:
done